about.md

```diff
@@ -0,0 +1,145 @@
+    control-your-metadata-with-yaml: Control Your Metadata With YAML
+  flexibility-and-customization:
+    _title: Flexibility and Customization
+    themes: Themes...
+    plugins: Plugins...
+  getting-started: Getting Started
+nav-url: /about/
+nav: 2
+---
+
+Pico is a stupidly simple, blazing fast, flat file CMS.  For those of us who aren't an expert webmaster, what does that even mean?  In the most basic sense, it means that there is no administration backend or database to deal with. You simply create .md files in the content folder and those files become your pages.  There's *much* more to Pico than that though.
+
+![Pico's Default Theme]({{ site.github.url }}/style/images/docs/pico_about_default_theme.png)
+Pico's default theme.  It's a bit bare... but that's intentional!
+
+* Center the text below images.
+
+Let's get this out of the way: Pico is *not* a turn-key solution.  If your goal is to deploy a generic-looking website without getting your hands dirty, Pico is probably not for you.
+
+* Is this too discouraging?
+
+That being said, Pico is incredibly extendable and customizable.  With a very small amount of configuration, you'll find yourself with a very personalized experience.  There's also a growing community creating ready-to-go [themes and plugins](#flexibility-and-customization) for Pico.  It is very possible to get started with Pico even with [no prior knowledge of HTML](#web-pages-without-html).  You'll find however that a little bit of coding knowledge will get you a long way.
+
+![NotePaper Theme for Pico]({{ site.github.url }}/style/images/docs/pico_about_notepaper_theme.jpg)
+NotePaper, a ready-to-go, community-developed theme.
+
+* Probably move this later, but I didn't want to forget to include it in the same commit as the image file.
+* Add example images for other themes.
+
+
+* insert some images showing Pico in use.  Borrow from readme?
+  * Can the site do thumbnails and lightboxing?
+* Add a style for these images? Their lack of a border or shadow makes them hard to distinguish from the page background.
+  * Again, can these be thumbnailed and lightboxed? That might work better.
+
+* Larger "About" portion at the beginning.
+  * Should contain all the "Why's" of Pico, before the "How's"
+
+Now let's dive into what makes Pico different from other solutions.
+
+## A Flat File CMS
+
+Pico is a Content Management System, or CMS.  If you've heard the term before, it's likely in the context of WordPress, Joomla, Drupal (intentionally not linking to the competition ;P) or many of the other popular CMS's on the block.  A Content Management System makes creating a beautiful and well-organized website easy and frustration free.  It can also help those without extensive knowledge of programming on the web to achieve the same professional quality website as those that do.
+
+ * Last sentence is a bit rough....
+ * Add headers for CMS and Flat file or is that excessive?
```

Comment:
Good point. Now I'm not so sure about it anymore :smile: I think you should probably do it the way *you* think it's better.